src/tests/test_field.py

```diff
@@ -0,0 +1,33 @@
+from django.test import TestCase
+from tests.models import DjangoIntervalTestModel
+import django
+from datetime import date as dt
+
+django.setup()
+
+
+class FuzzyDateParserFieldTest(TestCase):
```

Comment:
Can we either add a similar test for the regex field, or remove it from the model? I would opt for adding similar basic tests for the regex field.